.goreleaser.yml

```diff
@@ -14,7 +14,7 @@ builds:
   flags:
     - -trimpath
   ldflags:
-    - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+    - -s -w -X version.ProviderVersion={{.Version}}
```

Comment:
compare to https://github.com/hashicorp/terraform-provider-aws/search?q=ProviderVersion

We do not have a main.version or main.commit variable defined. We do now have a version.ProviderVersion.

Reply:
:+1: